### PR TITLE
support expression in update configs

### PR DESCRIPTION
**Pull request by critical27**
1. Support expression in update configs, e.g. 
`update configs storage:rocksdb_column_family_options={
    target_file_size_base=100*1024*1024
}`
2. Temporarily forbid modify `write_buffer_size` in `update configs`, it would cause rocksdb crash. Remove some configs which can't be set dynamically.
3. Remove some unused Status.
4. Support some mutable configs in console, such as `minlevel`, `v`, etc.

### CODE REVIEW

File: src/parser/MutateSentences.cpp [hunk 1]
```diff
@@ -162,6 +162,22 @@ std::string UpdateItem::toString() const {
...
+    buf += "=";
+    auto ret = value_->eval();
+    if (!ret.ok()) {
+        return ret.status();
+    }
+    if (Expression::isString(ret.value())) {
+        return Status::Error("Invalid string config value");
+    }
+
+    buf += Expression::toString(ret.value());
```
Comment (bright-starry-sky) [inline src/parser/MutateSentences.cpp]: Like this ！

**Comment by whitewum** [comment]
Changing the log level is the most frequently used command. Definitely, we should provide a document about this operation.

could you give an example doc. So I can abandon #1235

**Comment by critical27** [comment]
> Changing the log level is the most frequently used command. Definitely, we should provide a document about this operation.
> 
> could you give an example doc. So I can abandon #1235

Maybe we can still keep #1235, because if user have run register configs of `minloglevel` before, it is immutable, we still can't update it in console unless clear data on meta.

Anyway, I'll give a brief sample of update `minloglevel` in console to @Amber1990Zhang

**Comment by critical27** [comment]
Jenkins go

**Pull request merged**